src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/Email
+spring.datasource.username=postgres
```

Comment:
Do not push any creds to a remote repository but rather pass an environment variable here, eg ${DB_USERNAME}, env variables can be set in your IDEA configuration